src/js/controllers/chatController.js

```diff
@@ -76,25 +77,77 @@ module.exports = ViewController.extend({
             conversationDeferred.resolve(this.conversation);
         }
 
-        conversationDeferred.then(function(conversation) {
-            conversation.get('messages').create({
-                authorId: endpoint.appUserId,
-                text: text
-            }, {
-                success: messageDeferred.resolve,
-                error: messageDeferred.reject
-            });
 
-        });
+        conversationDeferred.then(_.bind(function(conversation) {
+            // update the user before sending the message to ensure properties are correct
+            this.user.save({}, {
```

Comment:
Please make sure this doesn't cause regression on first user message disappearing when SupportKit is initialized with only the appToken